### DISCUSSION TITLE
question in submission format

### DISCUSSION THREAD

**Discussion by IllgamhoDuck** [duchallenge-team/dude#9] [Q&A] [answered]
I saw submission format was written like the following.

```
{"questionId": "82889d7207d362cf89f620de626e30d8_49e9d9ef0e12d9a406660084d01db784",
"answer": "DUDE!",
"answer_confidence": 0,
"answer_abstain": false
}
```

but how could we answer the list type question with just str like "DUDE!"?
doesn't it should have to be submitted like the following with list wrapped?
```
{"questionId": "82889d7207d362cf89f620de626e30d8_49e9d9ef0e12d9a406660084d01db784",
"answer": ["DUDE!"],
"answer_confidence": 0,
"answer_abstain": false
}
```

so i submitted with the last one but not sure i did it correct because there is no score showing out which could tell me i submitted correct or not

**Comment by Jordy-VL** [comment, marked as answer]
The ground truth is provided as "answers" with type list, whereas predictions in submission should be under the key "answer", which can be of type string or list, depending on if the model predicted it to have multiple answers. In the evaluation code, this conversion is taken care of 😉. 

You will be provided the score after the competition closes :)

**Reply by IllgamhoDuck** [reply]
thanks! so even if I submitted a single answer like `answer: "duck"` as `answer: ["duck"]` it is not a problem?

**Reply by IllgamhoDuck** [reply]
@Jordy-VL I was not sure so just in case, I uploaded the same submission file with single answer as str and multi answer as list type

**Comment by Jordy-VL** [comment]
That's not a problem, I will take the best one into consideration ;)

FYI, do not forget to submit a method description (if you did not include
it with the submission).
